### ISSUE TITLE
Bad Handshake / SSL errors

### ISSUE THREAD

**Issue by wtluke** [Flexget/Flexget#1141]
Hi,

Been getting a lot of these while searching Kat for the last few days, on versions 2.0.9 - 2.0.11

`2016-05-05 12:03 WARNING  kat           kat-nores       Search resulted in: ("bad handshake: Error([('SSL routines', 'SSL23_GET_SERVER_HELLO', 'sslv3 alert illegal parameter')],)",)`

```  kat-nores:
    verify_ssl_certificates: no
    discover:
      what:
        - movie_list: allmyfilms
      from:
        - kat:
            category: movies
            verified: yes
        - torrentz:
            reputation: verified
    template: movietemplate
    priority: 9
    seen: local
    #assume_quality: 480p bluray h264 mp3
    quality: hdrip+
    #  - xvid+
    delay: 10 hours```

An issue with Kat, or a bug?

**Comment by liiight** [comment]
What version of python?

**Comment by wtluke** [comment]
2.7

**Comment by cvium** [comment]
2.7.what? Is it 2.7.3, 2.7.9 or 2.7.10 or what?

**Comment by wtluke** [comment]
Sorry. 2.7.9

**Comment by wtluke** [comment]
Just some updates on this. I've tried

`sudo pip install requests[security]`

It's all up-to-date. I have noticed that running the command manually with no-cache and discover-now avoids the errors *sometimes*. It's difficult to pinpoint when it's going to fail.

`/usr/local/bin/flexget -c /home/pi/flex/config.yml execute --discover-now --now --no-cache --tasks`

Via cron and also the Scheduler, it always results in the SSL error.

1) Renaming the task - this worked on the first run without errors. Subsequent runs produced the error
2) I replicated the task on another config and db, with a much smaller movie-list, and it almost always works. This is run manually from the command line.

I'm at a loss. Let me know if I can help with other commands or logs
